src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1332,14 +1332,16 @@ public String getFeedbackSessionResultSummaryAsCsv(String courseId,
                                                        String instructorEmail,
                                                        String filterText,
                                                        boolean isMissingResponsesShown,
-                                                       boolean isStatsShown)
+                                                       boolean isStatsShown,
+                                                       String questionId)
```

Comment:
It would be good if you can update the Method description on @param questionId that it is nullable, else might be confusing as the original method description requires non-null for every param